lib/csp.js

```diff
@@ -1,5 +1,23 @@
 'use strict';
 
+/**
+ * Parese Policy Rules
+ * @param {Object} policyRules
+ * @api private
+ */
+function parsePolicyRules(policyRules) {
+    if (Array.isArray(policyRules)) {
+        return policyRules.join('; ') + '; ';
+    } else {
+        var value = '', key;
+
+        for (key in policyRules) {
+            value += key + ' ' + policyRules[key] + '; ';
+        }
```

Comment:
@turboMaCk Ah, this is what I missed. I was thinking that this would be throwing an error. Thanks for explaining.